src/Process/ProcessUtils.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace GrumPHP\Process;
+
+/**
+ * ProcessUtils is a bunch of utility methods.
+ *
+ * This class contains static methods only and is not meant to be instantiated.
+ *
+ * @author Martin Hasoň <martin.hason@gmail.com>
+ */
+class ProcessUtils
```

Comment:
does it has to be exposed as public interface which interface is under SEMVER control?
if so, please at least make it final to not open it too widely
if not, please mark with `@internal`

Reply:
It does not. So I'll mark it both as `final` and `@internal`.